packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -107,10 +108,10 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             {
               borderWidth: 2,
               borderStyle: 'solid',
-              borderColor: theme.palette.neutralSecondary,
+              borderColor: semanticColors.smallInputBorder, // Could use focusBorder also but better to keep with the input theme
```

Comment:
It's not "better" but correct to stay with the input theme. So focus border shouldn't even be considered.